### ISSUE TITLE
Openswan obsoleted causing ash-linux and systemprep failures

### ISSUE THREAD

**Issue by lorengordon** [plus3it/ash-linux-formula#143]
Systemprep is failing when running against RedHat AMIs because openswan has been obsoleted by libreswan in the RHUI repos... The failed state file is cat3/V38687.sls.

```
# yum install openswan
Loaded plugins: amazon-id, product-id, rhui-lb, search-disabled-repos, subscription-manager
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Setting up Install Process
Package openswan-2.6.32-37.el6.x86_64 is obsoleted by libreswan-3.15-5.3.el6.x86_64 which is already installed
Nothing to do
```

**Comment by ferricoxide** [comment]
This is a Red Hat 6.8'ism. Need to add a grain to set the checked/installed *`swan` package base on the `osrelease`  grain (I _think_ that's the one that reports the `X.Y` - need to spin up an AMI and see what `grain.get` reports)

**Comment by ferricoxide** [comment]
Looks like `libreswan`s available as far back as the oldest CentOS 6 AMI I could find (6.3), so I'm going to drop the version-checking logic that the first fix attempts and just change the `openswan` install to install `libreswan`, instead.

**Comment by ferricoxide** [comment]
Closed by PR #148
